ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/AttestationValidator.java

```diff
@@ -113,7 +112,8 @@ private InternalValidationResult singleAttestationChecks(final Attestation attes
   }
 
   SafeFuture<InternalValidationResult> singleOrAggregateAttestationChecks(
-      final ValidateableAttestation validateableAttestation, final OptionalInt receivedOnSubnetId) {
+      final ValidateableAttestation validateableAttestation,
+      final Optional<Integer> receivedOnSubnetId) {
```

Comment:
Why did we switch from `OptionalInt` to `Optional<Integer>`?  `OptionalInt` is probably better if it's possible to stick with it.

Reply:
Reversed.